.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,68 @@
 <!--
 Thanks for sending a pull request!
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/master/CONTRIBUTING.md#submitting-pull-requests).
+Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
 
 In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.
+
+Finally, be sure to sign your commits (e.g. `git commit -s`)
+-->
+
+
+#### What type of PR is this?
+
+<!--
+Please label this pull request according to what type of issue you are
+addressing, especially if this is a release targeted pull request.
+
+Uncomment only one `/kind <>` line, hit enter to put that in a new line, and
+remove leading whitespace from that line:
 -->
+
+> /kind api-change
+> /kind bug
+> /kind cleanup
+> /kind deprecation
+> /kind design
+> /kind documentation
+> /kind failing-test
+> /kind feature
+> /kind flake
+> /kind other
+
+#### What this PR does / why we need it:
+
+#### How to verify it
+
+#### Which issue(s) this PR fixes:
+
+<!--
+Automatically closes linked issue when PR is merged.
+Uncomment the following comment block and include the issue
+number or None on one line.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`, or `None`.
+-->
+
+<!--
+Fixes #
+or
+None
```

Comment:
Has anyone ever done this `None` thing? If so why? It just seems like clutter; I vote to just say "Only include Fixes if the PR really fixes something"

Reply:
Also, please make it clear that `Fixes #` should also be in the commit message, for my sanity when writing release notes.

Reply:
I can, but...this is easily missed/forgotten, even if it's in the template.  I think that's what the labels are suppose to be used for (filtering).